crates/sui/src/client_commands.rs

```diff
@@ -1249,18 +1336,51 @@ impl SuiClientCommands {
                 type_args,
                 gas,
                 gas_budget,
+                dry_run,
                 gas_price,
                 args,
                 serialize_unsigned_transaction,
                 serialize_signed_transaction,
             } => {
-                let tx_data = construct_move_call_transaction(
-                    package, &module, &function, type_args, gas, gas_budget, gas_price, args,
-                    context,
+                let tx_kind = construct_move_call_transaction_kind(
```

Comment:
This function feels a bit redundant now -- it's just wrapping the call to the builder.